src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
shorthand begs to be here 🙂 
it's a common practice - so start applying the soonest )
```suggestion
    name,
    wheels,
    version,
```